dev-packages/e2e-tests/test-applications/angular-17/playwright.config.ts

```diff
@@ -29,8 +29,8 @@ const config: PlaywrightTestConfig = {
      */
     timeout: 10000,
   },
-  /* Run tests in files in parallel */
-  fullyParallel: true,
+  fullyParallel: false,
+  workers: 1,
   /* Fail the build on CI if you accidentally left test.only in the source code. */
```

Comment:
Just to avoid bleeds of events due to the proxy. 5 tests shouldn't make much difference here.